tests/api-key.spec.coffee

```diff
@@ -169,17 +169,6 @@ describe 'Request (api key):', ->
 							.get('query')
 							m.chai.expect(promise).to.eventually.be.null
 
-						it 'should not pass an apikey query string', ->
```

Comment:
It was exactly identical to the test above it, must've got duplicated at some point.